lib/features/base/mixin/message_dialog_action_mixin.dart

```diff
@@ -185,24 +191,29 @@ mixin MessageDialogActionMixin {
               imagePaths,
               listTextSpan: listTextSpan,
               titleActionButtonMaxLines: titleActionButtonMaxLines,
-              isArrangeActionButtonsVertical: isArrangeActionButtonsVertical
+              isArrangeActionButtonsVertical: isArrangeActionButtonsVertical,
+              useIconAsBasicLogo: useIconAsBasicLogo,
             )
             ..key(key)
             ..title(title ?? '')
             ..content(message)
             ..addIcon(icon)
-            ..paddingButton(paddingButton)
-            ..colorConfirmButton(actionButtonColor ?? AppColor.colorTextButton)
-            ..colorCancelButton(cancelButtonColor ?? AppColor.colorCancelButton)
-            ..marginIcon(icon != null ? const EdgeInsets.only(top: 24) : null)
-            ..paddingTitle(paddingTitle)
-            ..marginIcon(EdgeInsets.zero)
-            ..paddingContent(const EdgeInsets.only(left: 44, right: 44, bottom: 24, top: 12))
-            ..marginButton(hasCancelButton ? null : const EdgeInsets.only(bottom: 16, left: 44, right: 44))
-            ..styleTitle(titleStyle ?? const TextStyle(fontSize: 20, fontWeight: FontWeight.bold, color: Colors.black))
-            ..styleContent(messageStyle ?? const TextStyle(fontSize: 14, fontWeight: FontWeight.normal, color: AppColor.colorContentEmail))
-            ..styleTextCancelButton(cancelStyle ?? const TextStyle(fontSize: 17, fontWeight: FontWeight.w500, color: AppColor.colorTextButton))
-            ..styleTextConfirmButton(actionStyle ?? const TextStyle(fontSize: 17, fontWeight: FontWeight.w500, color: Colors.white))
+            ..colorConfirmButton(actionButtonColor ?? AppColor.blue700)
+            ..colorCancelButton(cancelButtonColor ?? AppColor.colorF3F6F9)
+            ..styleTitle(titleStyle)
+            ..styleContent(messageStyle)
+            ..styleTextCancelButton(cancelStyle ?? const TextStyle(
+              fontSize: 17,
+              height: 24 / 17,
+              fontWeight: FontWeight.w500,
+              color: AppColor.steelGray600,
+            ))
+            ..styleTextConfirmButton(actionStyle ?? const TextStyle(
+              fontSize: 17,
+              height: 24 / 17,
+              fontWeight: FontWeight.w500,
+              color: Colors.white,
+            ))
```

Comment:
we should use as the typography define in figma to make it easy to change in the future.
```suggestion
            ..styleTextConfirmButton(actionStyle ?? Theme.of(context).textTheme.bodyLarge))
```

<img width="519" alt="image" src="https://github.com/user-attachments/assets/a40aecdb-573b-4fe1-949c-4bf12179ee59" />

And define details of bodyLarge in TextTheme(), then use it with ThemeData

Reply:
```
static var fallbackTextTheme = const TextTheme(
    bodyLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: -0.15,
    ),
    bodyMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.25,
    ),
    bodySmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.4,
    ),
    labelLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.1,
    ),
    labelSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.5,
    ),
    displayLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w700,
    ),
    displayMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    displaySmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    headlineMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
    ),
    headlineSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    headlineLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      fontSize: 32,
    ),
    titleLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: -0.15,
    ),
    titleMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.15,
    ),
    titleSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.1,
    ),
  );
```

Reply:
Updated